### DISCUSSION TITLE
Heat Release Rate

### DISCUSSION THREAD

**Discussion by Alexg2991** [firemodels/fds#12713] [Q&A]
Hello, 

I'm trying to model a 2Mw Medium fire in a small residential unit (Approx 7*12). the door to the unit is open so the smoke can spill out, but obviously, it fills the room after a certain time (250s). the problem is, after the smoke completely fills the room, the HRR (attached) starts to diminish (around 450  onward). I have chosen a "Custom" Ramp-Up Time in which after 413s the fraction is 1, So I want to peaCk to remain at 2Mw. The reaction type is "Simple Chemistry" and the reaction is "SFPE POLYURETHANE"

What do you think is the problem of HRR not being able to maintain?
![Capture](https://github.com/firemodels/fds/assets/139024781/2d670b73-9a2b-4fae-ba97-37e9614c4f16)

Thank you

**Comment by drjfloyd** [comment]
Fuel requires mixing with oxygen inside the domain before it can burn.

**Comment by Alexg2991** [comment]
Thank you for your response, 

My understanding was that the solver would introduce as much needed oxygen into the domain to "keep" the HRR to my specified amount. So this is not the case. 

Is there any way I can introduce artificial oxygen into the domain so HRR can remain constant? (without adding makeup air)

**Comment by drjfloyd** [comment]
Please see the User's Guide and the Verification Guide for examples of the various types of boundary conditions available to you.

**Comment by Alexg2991** [comment]
I dont want to introduce makeup air into the domain using specific boundary conditions. Can you please be a bit more specific? It would be great if you can cite the relevant section from any of the guids.Much appreciated.

**Comment by drjfloyd** [comment]
If you want add oxygen somehow to the domain, then you need some input that tells FDS to add oxygen. That input is a boundary condition. You will need to figure out some boundary condition that makes sense for what you are trying to do that introduces oxygen.
